src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -41,6 +58,21 @@
     private static final String TAG_WORKER = "worker";
 
     /**
+     * NBTTag to store the recipes list.
+     */
+    private static final String TAG_RECIPES = "recipes";
+
+    /**
+     * Tag to serialize ITokens.
+     */
+    private static final String TAG_TOKEN = "tokenTag";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "TAG_TOKEN" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)